### PR TITLE
Add test for closing ws twice

### PR DESCRIPTION
**Pull request by bellini666**
When an error happens inside the handler I see a lot of `RuntimeError: Unexpected ASGI message 'websocket.close', after sending 'websocket.close'.` being raised by uvicorn: https://sentry.2u.app.br/share/issue/83c133f3792c4305b305f0ce31c8ad0c/

This usually happens because the init timeout handler calls `close` when another `close` has already being called (due to an error)

Are there any bad side effects for doing this?

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/), so make sure to pick the appropriate type. If in doubt feel free to ask :)


Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Thiago Bellini Ribeiro for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

File: tests/websockets/test_graphql_transport_ws.py [hunk 1]
```diff
@@ -156,6 +161,49 @@ async def test_connection_init_timeout_cancellation(
         assert ws.closed
 
 
+async def test_close_twice(
+    mocker: MockerFixture, request, http_client_class: Type[HttpClient]
+):
+    if sys.version_info < (3, 8):
+        pytest.skip("Task name was introduced in 3.8 and we need it for this test")
```
Comment (patrick91) [inline tests/websockets/test_graphql_transport_ws.py:168]: can we move this to a marker on the test? 

```python
@pytest.mark.skipif(...)
async def test_close_twice(
```

File: tests/websockets/test_graphql_transport_ws.py [hunk 2]
```diff
@@ -156,6 +161,49 @@ async def test_connection_init_timeout_cancellation(
...
+
+    if http_client_class == AioHttpClient:
+        pytest.skip(
+            "Closing a AIOHTTP WebSocket from a task currently doesnt work as expected"
```
Comment (patrick91) [inline tests/websockets/test_graphql_transport_ws.py:172]: how come, what happens there?

File: tests/websockets/test_graphql_transport_ws.py (hunk 1 above)
Comment (bellini666) [reply tests/websockets/test_graphql_transport_ws.py:168]: Sure! Will do that

File: tests/websockets/test_graphql_transport_ws.py (hunk 2 above)
Comment (bellini666) [reply tests/websockets/test_graphql_transport_ws.py:172]: Not really sure, I copied that skip from other tests (https://github.com/strawberry-graphql/strawberry/blob/main/tests/websockets/test_graphql_transport_ws.py#L103) that also test the timeout assuming that my test would have the same issue

I'll try without it though

Comment (bellini666) [reply tests/websockets/test_graphql_transport_ws.py]: I removed and it worked fine =P

**Comment by bellini666** [comment]
@patrick91 it seems that the timeout PR also fixed this issue. I changed it to instead of doing what I was doing to instead just add a test to make sure it will not break in the future.

I also removed the `RELEASE.md` because I don't think a test is enough to justify a new release

**Comment by kristjanvalur** [comment]
Yes, the timeout PR specifically made sure that there were no race conditions in closing the websocket connection.

**Pull request merged**